python/astro_metadata_translator/translators/sdss.py

```diff
@@ -183,8 +189,13 @@ def to_location(self) -> EarthLocation:
         location : `astropy.coordinates.EarthLocation`
             An object representing the location of the telescope.
         """
-        # Look up the value since files do not have location
-        value = EarthLocation.of_site("apo")
+        # Look up the value since files do not have location.
+        # This might require a network look up if the cached database
+        # is not accessible. If it fails fall back to hard-coded location.
+        try:
+            value = EarthLocation.of_site("apo")
+        except astropy.coordinates.UnknownSiteException:
```

Comment:
You either need to import `astropy.coordinates` or `from astropy.coordinates import UnknownSiteException` or this won't work (I checked by deleting my cache and running without internet).

Reply:
wow. How come none of the linters complained? VSCode was more than happy to add this...